<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KTodoItemInfo</class>
 <widget class="QWidget" name="KTodoItemInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>210</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>210</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KTodoItemInfo</string>
  </property>
  <widget class="QLabel" name="m_todocontent_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>16</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Microsoft YaHei;
text-align: left;</string>
   </property>
   <property name="text">
    <string>去山道健身房健身</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_proirity_box">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>43</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
font-family: Microsoft YaHei;
text-align: left;</string>
   </property>
   <item>
    <property name="text">
     <string>无优先级</string>
    </property>
    <property name="icon">
     <iconset resource="kmainwindow.qrc">
      <normaloff>:/todoitem/icons/priority_null.png</normaloff>:/todoitem/icons/priority_null.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>低优先级</string>
    </property>
    <property name="icon">
     <iconset resource="kmainwindow.qrc">
      <normaloff>:/todoitem/icons/priority_low.png</normaloff>:/todoitem/icons/priority_low.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中优先级</string>
    </property>
    <property name="icon">
     <iconset resource="kmainwindow.qrc">
      <normaloff>:/todoitem/icons/priority_mid.png</normaloff>:/todoitem/icons/priority_mid.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>高优先级</string>
    </property>
    <property name="icon">
     <iconset resource="kmainwindow.qrc">
      <normaloff>:/todoitem/icons/priority_high.png</normaloff>:/todoitem/icons/priority_high.png</iconset>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="split_line1">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>72</y>
     <width>206</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(169, 169, 169);
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_date_btn">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>80</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left;
font-family: Microsoft YaHei;</string>
   </property>
   <property name="text">
    <string>07月30日到期</string>
   </property>
   <property name="icon">
    <iconset resource="kmainwindow.qrc">
     <normaloff>:/todoiteminfo/icons/calendar.png</normaloff>:/todoiteminfo/icons/calendar.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="m_date_info_label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>114</y>
     <width>181</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Microsoft YaHei;
text-align: left;</string>
   </property>
   <property name="text">
    <string>每天 重复待办</string>
   </property>
  </widget>
  <widget class="QPushButton" name="split_line_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>140</y>
     <width>196</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(189, 190, 189);
background-color: rgb(233, 233, 233);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_clock_btn">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>155</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left;
font-family: Microsoft YaHei;</string>
   </property>
   <property name="text">
    <string>07月30日 09:10提醒我</string>
   </property>
   <property name="icon">
    <iconset resource="kmainwindow.qrc">
     <normaloff>:/todoiteminfo/icons/clock.png</normaloff>:/todoiteminfo/icons/clock.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="split_line_3">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>195</y>
     <width>196</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(233, 233, 233);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_add_description_btn">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>210</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left;
font-family: Microsoft YaHei;</string>
   </property>
   <property name="text">
    <string>添加描述</string>
   </property>
   <property name="icon">
    <iconset resource="kmainwindow.qrc">
     <normaloff>:/todoiteminfo/icons/description.png</normaloff>:/todoiteminfo/icons/description.png</iconset>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_description_edit">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>244</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left;
font-family: Microsoft YaHei;</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>点击上方“添加描述”对待办事项添加描述</string>
   </property>
  </widget>
  <widget class="QPushButton" name="split_line_4">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>307</y>
     <width>196</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(233, 233, 233);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_location_edit">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>322</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 225, 225);
border: none;
text-align: left;
font-family: Microsoft YaHei;</string>
   </property>
   <property name="placeholderText">
    <string>添加地点</string>
   </property>
  </widget>
  <widget class="QPushButton" name="split_line_5">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>353</y>
     <width>196</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(233, 233, 233);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_add_remark_btn">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>366</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left;
font-family: Microsoft YaHei;</string>
   </property>
   <property name="text">
    <string>添加备注</string>
   </property>
   <property name="icon">
    <iconset resource="kmainwindow.qrc">
     <normaloff>:/todoiteminfo/icons/remark.png</normaloff>:/todoiteminfo/icons/remark.png</iconset>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_remark_edit">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>400</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left;
font-family: Microsoft YaHei;</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>点击上方“添加备注”对待办事项添加备注</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_is_check_box">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>16</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_save_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>499</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="m_datetime_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="kmainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
